scripts/check_tx_same_sepolia_mainnet3.py

```
@@ -0,0 +1,171 @@
+        print(f"\nParsing {network1_info['name']} transaction: {tx1_hash}")
+        network1_result = self.parse_transaction(tx1_hash, 'sepoliadev1')
+        
+        print(f"\nParsing {network2_info['name']} transaction: {tx2_hash}")
+        network2_result = self.parse_transaction(tx2_hash, 'sepoliadev2')
+        
+        if not network1_result:
+            return f"Error: Could not parse {network1_info['name']} transaction {tx1_hash}"
+        if not network2_result:
+            return f"Error: Could not parse {network2_info['name']} transaction {tx2_hash}"
+            
+        comparison = f"Transaction Comparison\n"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)